sigar_util.go

```diff
@@ -3,20 +3,21 @@
 package gosigar
 
 import (
-	"unsafe"
+	"bytes"
 )
 
-// TODO (2020-07-14): Fix the unsafe pointer conversion. https://github.com/elastic/gosigar/issues/138
-//go:nocheckptr
-func bytePtrToString(ptr *int8) string {
-	bytes := (*[10000]byte)(unsafe.Pointer(ptr))
+// byteListToString converts the raw byte arrays we get into a string. This is a bit of a process, as byte strings are normally []uint8
+func byteListToString(raw []int8) string {
+	byteList := make([]byte, len(raw))
```

Comment:
Consider using `bytes.IndexByte` to precompute necessary length to minimize allocations.

Reply:
Nevermind, that's an []int8 and not a []byte.